src/jmh/java/column/store/query/span/FilterForTraceId.java

```diff
@@ -0,0 +1,100 @@
+package column.store.query.span;
+
+import column.store.Utils;
+import column.store.api.column.Column;
+import column.store.api.column.IdColumn;
+import column.store.api.query.Query;
+import column.store.api.read.Reader;
+import static column.store.api.query.Filter.whereId;
+import static column.store.util.Conditions.checkState;
+
+import java.io.IOException;
+import java.nio.file.Path;
+
+import org.openjdk.jmh.annotations.Benchmark;
+import org.openjdk.jmh.annotations.BenchmarkMode;
+import org.openjdk.jmh.annotations.Fork;
+import org.openjdk.jmh.annotations.Level;
+import org.openjdk.jmh.annotations.Measurement;
+import org.openjdk.jmh.annotations.Mode;
+import org.openjdk.jmh.annotations.Param;
+import org.openjdk.jmh.annotations.Scope;
+import org.openjdk.jmh.annotations.Setup;
+import org.openjdk.jmh.annotations.State;
+import org.openjdk.jmh.annotations.Warmup;
+import org.openjdk.jmh.infra.Blackhole;
+
+@Measurement(iterations = 3)
```

Comment:
in the final version of all benchmark: should agree on common warmup iterations, forks, ...